### PR TITLE
UX fixes for ext:update

### PR DESCRIPTION
**Pull request by joehan**
### Description
Fixes a few UX issues we found with ext:update.
- Consent prompts now default to yes.
- Cloud Function resources are now displayed correctly.
- Resources won't be shown as deleted when only their description changes.
	 
### Scenarios Tested
Screen Shot 2019-12-05 at 11 37 08 AM

### CODE REVIEW

File: src/extensions/updateHelper.ts [hunk 1]
```diff
@@ -134,17 +134,21 @@ export async function displayChangesRequiringConfirmation(
...
+    ? `${resource.name} (Cloud Function): ${resource.description}\n`
     : `${resource.name} (${resource.type})\n`;
 }
 
 async function getConsent(field: string, message: string): Promise<void> {
   const consent = await promptOnce({
     type: "confirm",
     message,
-    default: false,
+    default: true,
```
Comment (joehan) [inline src/extensions/updateHelper.ts]: Internal bug 145691993

File: src/extensions/updateHelper.ts [hunk 2]
```diff
@@ -134,17 +134,21 @@ export async function displayChangesRequiringConfirmation(
...
 }
 
+function compareResources(resource1: extensionsApi.Resource, resource2: extensionsApi.Resource) {
+  return resource1.name == resource2.name && resource1.type == resource2.type;
+}
+
 function getResourceReadableName(resource: extensionsApi.Resource): string {
-  return resource.type === "function"
-    ? `${resource.name} (${resource.description})\n`
+  return resource.type === "firebaseextensions.v1beta.function"
```
Comment (joehan) [inline src/extensions/updateHelper.ts]: Internal bug 145691513

File: src/extensions/updateHelper.ts [hunk 3]
```diff
@@ -93,12 +93,12 @@ export async function displayChangesRequiringConfirmation(
   const resourcesDiffDeletions = _.differenceWith(
     spec.resources,
     _.get(newSpec, "resources", []),
-    _.isEqual
+    compareResources
```
Comment (joehan) [inline src/extensions/updateHelper.ts]: internal bug 145695489

**Comment by bkendall** [comment]
Looks like a unit test is broken too.

File: CHANGELOG.md [hunk 4]
```diff
@@ -1 +1,2 @@
 * Fixes an issue where all multi-site pre- and post-deploy hooks trigger for targeted deploys (#1160).
+* Fixes UX bugs with ext:update.
```
Comment (bkendall) [inline CHANGELOG.md]:
```suggestion
* Fixes UX bugs in `ext:update`.
```

**Pull request merged**